### ISSUE TITLE
Autocompletion

### ISSUE THREAD

**Issue by a-b** [klauscfhq/taskbook#11]
**Is your feature request related to a problem? Please describe.**
I want shell to autocomplete.

**Describe the solution you'd like**
Implement and install autocompletion.

**Comment by danielcooperxyz** [comment]
This would be nice. What would be autocompleted? Command args? Board names?

Board names is something that would be most useful I think.

Would a package like [omelette](https://www.npmjs.com/package/omelette) help with this?

**Comment by rjoydip** [comment]
Yeah, I think [omelette](https://www.npmjs.com/package/omelette) would be good for this feature.
